scripts/Costs.py

```diff
@@ -321,6 +321,7 @@ def tradeCosts(costType, regions, years):
             if costType[0] == 'Variable O&M':
                 for year in years:
                     data.append([region,tech,1,year,trnCost])
+                    data.append([region,tech,2,year,trnCost])

```

Comment:
MODE_OF_OPERATION is used in several places, and I think it could really use a definition directly in the code.

Reply:
Created issue